test/qtiCommonRenderer/interactions/textEntry/test.js

```diff
@@ -33,6 +33,7 @@ define([
         if (instance && instance.popperInstance.popper) {
             return $(instance.popperInstance.popper);
         }
+        return instance
```

Comment:
Now it returns either a `jQuery` object (the original return) or a `string`.
Could you please make the method return a homogeneous type?